#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_spanish_translation.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Closes: #399176.

@DPATCH@
diff -urNad dansguardian-2.8.0.6-antivirus-6.4.4.1~/languages/spanish/messages dansguardian-2.8.0.6-antivirus-6.4.4.1/languages/spanish/messages
--- dansguardian-2.8.0.6-antivirus-6.4.4.1~/languages/spanish/messages	2005-08-14 03:39:35.000000000 +0200
+++ dansguardian-2.8.0.6-antivirus-6.4.4.1/languages/spanish/messages	2006-12-04 21:57:19.000000000 +0100
@@ -16,7 +16,7 @@
 "400","Combinación de frases no permitida: "
 "401","Combinación de frases no permitida."
 "402","Límite de ponderación de frases de "
-"403", Límite de ponderación de frases excedido"
+"403", "Límite de ponderación de frases excedido"
 
 "500","Sitio no permitido: "
 "501","URL no permitida: "
